### PR TITLE
Adds hot module share copy

### LINKED ISSUE #4637: Hot phase share copy
need a field in https://www.dosomething.org/admin/config/dosomething/dosomething_campaign for what the share prompt should be during the hot phase

Suggested field name "Hot Phase Share Prompt".

### PR DESCRIPTION
**Pull request by itsjoekent**
Fixes #4637 

Sorry for all of the files @sbsmith86 recently added being in commit 59371d1, had to fix a merge conflict and I guess everything decided to follow along for the ride. 

screen shot 2015-07-02 at 2 45 25 pm

@aaronschachter can you review?

### CODE REVIEW

File: lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc [hunk 1]
```diff
@@ -201,6 +201,11 @@ function dosomething_campaign_preprocess_action_page(&$vars, &$wrapper) {
     $vars['tumblr_link'] = dosomething_helpers_get_tumblr_intent($tumblr_options);
   }
 
+  $vars['hot_module_enabled'] = variable_get('dosomething_campaign_enable_hot_module', FALSE);
+  $vars['hot_module_share_copy'] = variable_get('dosomething_campaign_share_copy', 'Rally your friends to crush this goal!');
```
Comment (aaronschachter) [inline lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc]: Shouldn't this need a `t()` ?

File: lib/modules/dosomething/dosomething_campaign/dosomething_campaign.admin.inc [hunk 2]
```diff
@@ -276,6 +276,13 @@ function dosomething_campaign_admin_config_form($form, &$form_state) {
     '#description' => t("If set, hot module functionaliy will be turned on."),
```
Comment (aaronschachter) [inline lib/modules/dosomething/dosomething_campaign/dosomething_campaign.admin.inc]: Mind fixing this typo while you're in here?

File: lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc (hunk 1 above)
Comment (itsjoekent) [inline lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc]: yep!

**Comment by aaronschachter** [comment]
Make minor changes then ok to merge :+1:

File: lib/modules/dosomething/dosomething_campaign/dosomething_campaign.admin.inc (hunk 2 above)
Comment (itsjoekent) [inline lib/modules/dosomething/dosomething_campaign/dosomething_campaign.admin.inc]: It took me a minute to spot that typo.

**Pull request merged**